src/utils/api/notice.ts

```diff
@@ -1,7 +1,24 @@
 import { Paginated } from '@beabee/beabee-common';
 import { deserializeDate } from '.';
+
 import axios from '../../lib/axios';
-import { GetNoticeData, GetNoticesQuery, Serial } from './api.interface';
+import {
+  CreateNoticeData,
+  GetNoticeData,
+  GetNoticesQuery,
+  Serial,
+} from './api.interface';
+
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+function deserializeNotice(notice: any): any {
```

Comment:
I don't think this needs to use type `any`, it could be
```ts
function deserializeNotice(notice: Serial<GetNoticeData>): GetNoticeData {
  ...
}
```

The idea is that the compiler will throw an error if you forget to deserialize a property, e.g.
```ts
function deserializeNotice(notice: Serial<GetNoticeData>): GetNoticeData {
  return notice;
  // Type 'Serial<GetNoticeData>' is not assignable to type 'GetNoticeData'.
  //   Types of property 'createdAt' are incompatible.
  //     Type 'string' is not assignable to type 'Date'.ts(2322)
}
```

This is especially useful in the future as we could for example add a new `Date` property to `GetNoticeData`, then the compiler would automatically catch that we need to deserialize the date.

At the moment there are a few `any` deserializers (e.g. `deserializeContact`), but that's just because I haven't figured out how to write proper generic types for them to work.